Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,1006 @@
+        a crop variant for "mobile" and "desktop", or different aspect ratios,
+        or any custom scheme.
+
+    ..  _crowdin:
+    -   *Crowdin*
+
+        Crowdin is a translation tool used for localizing and translating TYPO3
+        content into different languages.
+
+    ..  _ctype:
+    -   *CType*
+
+        :sql:`CType` stands for Content Type and is a database column field of
+        the vital "tt_content" database table, where all content elements are
+        stored. This column defines the name of a specific element, and
+        influences how it is presented in the backend and frontend.
+
+*   D
+
+    ..  _dashboard---widgets:
+    -   *Dashboard / Widgets*
+
+        The dashboard is a customizable start page in the TYPO3 backend that
+        contains various widgets for displaying important information and quick
+        access.
+
+    ..  _data-processor:
+    -   *Data Processor*
+
+        A data processor is a component that processes and manipulates data
+        before it is displayed in the TYPO3 frontend. Data processors are
+        implemented as PHP code and can be executed through TypoScript
+        configuration, and affect data passed on to Fluid templates. Thus, it is
+        a specific implementation to operate on data before it is passed on to
+        the presentation layer (templates).
+
+    ..  _data-provider:
+    -   *Data Provider*
+
+        A data provider is a source of data that can be used by other components
+        within the TYPO3 system. Data providers are commonly used for passing on
+        data in the backend (for example defining available icons, item keys and
+        values).
+
+    ..  _datahandler:
+    -   *DataHandler*
+
+        The DataHandler is a central component in TYPO3 responsible for
+        processing and storing data changes. It is a large PHP class that is
+        used in the backend to receive data from the FormEngine (content
+        elements and records), and is also part of an extensive API that can be
+        used by extensions to operate on records.
+
+    ..  _db-analyzer---db-compare:
+    -   *DB Analyzer / DB Compare*
+
+        DB Analyzer and DB Compare are tools in TYPO3 that analyze and compare
+        database structures to identify needed changes on the database-level for
+        both for system upgrades and extension integration. Other systems often
+        call this "database migration".
+
+    ..  _db-mounts---mount-points:
+    -   *DB Mounts / Mount Points*
+
+        Mount points allow TYPO3 editors to mount a page (and its subpages) from
+        a different area in the current backend page tree.
+
+    ..  _dbal:
+    -   *DBAL*
+
+        The Database Abstraction Layer (DBAL) is an abstraction layer in TYPO3
+        that facilitates access to various database systems.
+
+*   E
+
+    ..  _enhancer:
+    -   *Enhancer*
+
+        An enhancer is a component that adds additional functionalities or
```

Comment:
I have search whole TYPO3 v13. The word "enhancer" is only used by the router. So `adds...improvements to existing TYPO3 featureS" is not correct.